File: 백준/Bronze/14659. 한조서열정리하고옴ㅋㅋ/한조서열정리하고옴ㅋㅋ.py
N = int(input())
H = list(map(int, input().split()))

res = H[0]
cnt = 0
maxcnt = 0

for i in range(1, len(H)):
    if H[i] > res:
        res = H[i]

        if maxcnt < cnt:
            maxcnt = cnt

        cnt = 0
        continue
    cnt += 1

if maxcnt < cnt:
    maxcnt = cnt

print(maxcnt)

File: 백준/Bronze/14659. 한조서열정리하고옴ㅋㅋ/한조서열정리하고옴ㅋㅋ.cc
#include <iostream>
#include <vector>
#define endl "\n"
using namespace std;

int main()
{
    int N;
    cin >> N;

    vector<int> H(N);
    for (int i = 0; i < N; i++)
    {
        cin >> H[i];
    }

    int res = H[0], cnt = 0, maxcnt = 0;
    for (int i = 1; i < N; i++)
    {
        if (H[i] > res)
        {
            res = H[i];
            if (cnt > maxcnt)
            {
                maxcnt = cnt;
            }
            cnt = 0;
            continue;
        }
        cnt++;
    }

    if (cnt > maxcnt)
    {
        maxcnt = cnt;
    }

    cout << maxcnt;
}